pkg/apis/kai/v1/binder/binder.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2025 NVIDIA CORPORATION
+// SPDX-License-Identifier: Apache-2.0
+
+// +kubebuilder:object:generate:=true
+package binder
+
+import (
+	"k8s.io/utils/ptr"
+
+	"github.com/NVIDIA/KAI-scheduler/pkg/apis/kai/v1/common"
+	"github.com/NVIDIA/KAI-scheduler/pkg/common/constants"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+const (
+	imageName                           = "binder"
+	defaultResourceReservationImageName = "resource-reservation"
+)
+
+type Binder struct {
+	Service *common.Service `json:"service,omitempty"`
+
+	// Webhook defines configuration for the binder service
+	// +kubebuilder:validation:Optional
+	Webhook *Webhook `json:"webhook,omitempty"`
```

Comment:
Does the binder still needs a webhook config after the kai-admission service?

Reply:
:+1: right it doesn't need it!